guides/source/documents.yaml

```diff
@@ -72,6 +72,10 @@
       url: active_support_core_extensions.html
       description: This guide documents the Ruby core extensions defined in Active Support.
     -
+      name: Active Support On Load Hooks
```

Comment:
> "Rails is responsible for what is loaded when, you should not load frameworks by yourself because that violates the contract, on_load is the API to integrate with the Rails boot process".

Nice, yea, I can add that tone no problem :+1: